packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -36,15 +36,13 @@ import {
 } from '../keys/checkKey.js';
 import { generateCollectionPairEntries } from '../keys/keycollection-operators.js';
 
-import './internal-types.js';
-
 /// <reference types="ses"/>
 
 const { quote: q, bare: b, details: X, Fail } = assert;
 const { entries, values } = Object;
 const { ownKeys } = Reflect;
 
-/** @type {WeakSet<Pattern>} */
+/** @type {WeakSet<import('./internal-types.js').Pattern>} */
```

Comment:
I see you were trying to avoid intrusive changes, but this is too much verbose noise.  Can you please rename `./internal-types.js` to `./types.js` to follow the same filename convention as `eventual-send`  (internal=types.{js,d.ts}, external=exports.{js,d.ts})?

Reply:
Sure, I can line this up in this change with @erights’s blessing. It’ll be a separate commit.

Reply:
I do not foresee an objection, so I’ve merged for expedience. Do let me know if I should revert the refactor of internal-types.js to merely types.js.